config/initializers/spree.rb

```diff
@@ -20,6 +20,12 @@
   # Auto-capture payments. Without this option, payments must be manually captured in the paypal interface.
   config.auto_capture = true
   #config.override_actionmailer_config = false
+
+  # S3 settings
+  config.s3_bucket = ENV['S3_BUCKET'] if ENV['S3_BUCKET']
```

Comment:
It only updates the stored preference on the server if `S3_BUCKET` is set in `application.yml`, and the corresponding update in ofn-install only sets that if the new optional `s3_images_bucket` key is set in the secrets file (which nobody has set yet, because it's new), so this will not make any changes on current systems that have been configured manually.

See: https://github.com/openfoodfoundation/ofn-install/pull/386/files